cmake/modules/hunter_create_cache_meta_directory.cmake

```diff
@@ -30,7 +30,7 @@ function(hunter_create_cache_meta_directory cache_directory result)
   file(COPY "${toolchain_info}" DESTINATION "${cache_meta_dir}")
 
   # Save package name and version
-  set(cache_meta_dir "${cache_meta_dir}/${HUNTER_PACKAGE_NAME}")
+  set(cache_meta_dir "${cache_meta_dir}/${HUNTER_PACKAGE_NAME}/${HUNTER_${HUNTER_PACKAGE_NAME}_RECIPE_HASH}")
```

Comment:
This will break cache feature. For example we have `Foo 1.0` in `hunter.cmake` and save it to cache, now we are adding `Foo 2.0` to `hunter.cmake`. Since `hunter.cmake` change => `*_RECIPE_HASH` change => `cache_meta_dir` changed => cache miss.

More info about cache mechanism may be found here:
* https://github.com/ruslo/hunter/wiki/dev.modules#hunter_download
* https://github.com/ruslo/hunter/issues/74
* https://github.com/ruslo/hunter/issues/194